### PR TITLE
Added tests for SMS limit

### PR DESCRIPTION
**Pull request by SerhiiDielieu**
Added tests for SMS limit
refactoring
added TestNG config file
added tests for Enrollment

### CODE REVIEW

File: src/main/java/enrollment/EndPointsEnrollment.java [hunk 1]
```diff
@@ -16,4 +16,70 @@
     public String getUrl() {
         return IRIS_BASE_URL.getUrl() + url;
     }
+
+    public static String getTemplate() {
+        return template;
+    }
+
+    private final static String template = "GhV+xAMAAgIBAQBgHuB6" +
```
Comment (adamsabel) [inline src/main/java/enrollment/EndPointsEnrollment.java]: This template format is not valid anymore

File: src/main/java/enrollment/EndPointsEnrollment.java [hunk 2]
```diff
@@ -16,4 +16,70 @@
     public String getUrl() {
         return IRIS_BASE_URL.getUrl() + url;
     }
+
+    public static String getTemplate() {
```
Comment (adamsabel) [inline src/main/java/enrollment/EndPointsEnrollment.java]: use @ Getter

File: src/test/java/badrequest/ContentTypesTests.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import static enrollment.EndPointsEnrollment.PRECHECK;
+import static io.restassured.RestAssured.given;
+import static org.apache.http.HttpStatus.SC_UNSUPPORTED_MEDIA_TYPE;
+import static otp.EndPointsOtp.OTP;
+import static otp.EndPointsOtp.VALIDATE;
+import static sms.EndPointsSms.SMS;
+import static wallet.EndPointsWallet.ADD_TO_WALLET;
+
+public class ContentTypesTests {
+    private final List<String> endpoints = List.of(OTP.getUrl(), VALIDATE.getUrl(), SMS.getUrl(), ADD_TO_WALLET.getUrl(), ENROLLMENT.getUrl(), PRECHECK.getUrl());
```
Comment (adamsabel) [inline src/test/java/badrequest/ContentTypesTests.java]: This list should be public static final in some enum that aggregates all endpoints. Looks like endpoint enums need refactoring

File: src/test/java/badrequest/ContentTypesTests.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    private final List<ContentType> contentTypes = List.of(ContentType.XML, ContentType.BINARY, ContentType.HTML, ContentType.TEXT, ContentType.URLENC, ContentType.ANY);
```
Comment (adamsabel) [inline src/test/java/badrequest/ContentTypesTests.java]: I think Arrays.asList(ContentType.values()) should do the trick

File: src/main/java/enrollment/EndPointsEnrollment.java (hunk 1 above)
Comment (SerhiiDielieu) [reply src/main/java/enrollment/EndPointsEnrollment.java]: fixed

File: src/main/java/enrollment/EndPointsEnrollment.java (hunk 2 above)
Comment (SerhiiDielieu) [reply src/main/java/enrollment/EndPointsEnrollment.java]: done

**Comment by SerhiiDielieu** [comment]
@adamsabel

File: src/test/java/badrequest/NotAllowedMethodsTests.java [hunk 5]
```diff
@@ -1,64 +1,54 @@
...
+import static enrollment.EndPointsEnrollment.ENROLLMENT;
+import static enrollment.EndPointsEnrollment.PRECHECK;
+import static io.restassured.RestAssured.given;
+import static otp.EndPointsOtp.OTP;
+import static otp.EndPointsOtp.VALIDATE;
+import static sms.EndPointsSms.SMS;
 import static org.apache.http.HttpStatus.SC_METHOD_NOT_ALLOWED;
+import static wallet.EndPointsWallet.ADD_TO_WALLET;
+
 @AllArgsConstructor
```
Comment (adamsabel) [inline src/test/java/badrequest/NotAllowedMethodsTests.java]: Why your tests have constructor?

File: src/main/java/enrollment/EndPointsEnrollment.java [hunk 6]
```diff
@@ -13,7 +14,5 @@
 
     private final String url;
 
-    public String getUrl() {
-        return IRIS_BASE_URL.getUrl() + url;
-    }
+    private final static String template = "GhV+xAMAAgIBAQBgHuB6" ;
```
Comment (adamsabel) [inline src/main/java/enrollment/EndPointsEnrollment.java]: This should be public - you are pushing changes to prod branch and they even dont compile

File: src/test/java/badrequest/NotAllowedMethodsTests.java (hunk 5 above)
Comment (SerhiiDielieu) [reply src/test/java/badrequest/NotAllowedMethodsTests.java]: they don;t

File: src/main/java/enrollment/EndPointsEnrollment.java (hunk 6 above)
Comment (SerhiiDielieu) [reply src/main/java/enrollment/EndPointsEnrollment.java]: done

**Comment by SerhiiDielieu** [comment]
@adamsabel

**Pull request merged**